llvm/test/Transforms/PGOProfile/memprof_undrift_missing_leaf.ll

```diff
@@ -0,0 +1,97 @@
+;; Tests memprof undrifting when the leaf frame is missing in the profile.
+;; This test case is taken from memprof_missing_leaf.ll with the profile
+;; drifted.
+
+;; Avoid failures on big-endian systems that can't read the profile properly
+; REQUIRES: x86_64-linux
```

Comment:
We don't need this anymore since we don't use binary profiles for testing after introducing the text format  :)

Reply:
Removed.